[2023-03-17 00:06:04,349] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: reto_tecnico_everst.download_data manual__2023-03-17T00:06:01.068320+00:00 [queued]>
[2023-03-17 00:06:04,357] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: reto_tecnico_everst.download_data manual__2023-03-17T00:06:01.068320+00:00 [queued]>
[2023-03-17 00:06:04,358] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-17 00:06:04,359] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-03-17 00:06:04,359] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-17 00:06:04,410] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): download_data> on 2023-03-17 00:06:01.068320+00:00
[2023-03-17 00:06:04,418] {standard_task_runner.py:52} INFO - Started process 212 to run task
[2023-03-17 00:06:04,421] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'reto_tecnico_everst', 'download_data', 'manual__2023-03-17T00:06:01.068320+00:00', '--job-id', '120', '--raw', '--subdir', 'DAGS_FOLDER/reto_tecnico_everst.py', '--cfg-path', '/tmp/tmpatdkzq68', '--error-file', '/tmp/tmpnfeyc2z9']
[2023-03-17 00:06:04,422] {standard_task_runner.py:80} INFO - Job 120: Subtask download_data
[2023-03-17 00:06:04,547] {task_command.py:369} INFO - Running <TaskInstance: reto_tecnico_everst.download_data manual__2023-03-17T00:06:01.068320+00:00 [running]> on host 7d9c3e4a2f58
[2023-03-17 00:06:04,756] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=reto_tecnico_everst
AIRFLOW_CTX_TASK_ID=download_data
AIRFLOW_CTX_EXECUTION_DATE=2023-03-17T00:06:01.068320+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-17T00:06:01.068320+00:00
[2023-03-17 00:06:08,295] {reto_tecnico_everst.py:38} INFO - Generando descarga del GZ
[2023-03-17 00:06:10,836] {reto_tecnico_everst.py:45} INFO - Guardando la base generada desde el API, path: dags/data/Raw/base_api/base_20230317_00.06.json
[2023-03-17 00:06:25,109] {python.py:173} INFO - Done. Returned value was: None
[2023-03-17 00:06:25,780] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=reto_tecnico_everst, task_id=download_data, execution_date=20230317T000601, start_date=20230317T000604, end_date=20230317T000625
[2023-03-17 00:06:27,900] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-17 00:06:29,062] {local_task_job.py:273} INFO - 2 downstream tasks scheduled from follow-on schedule check
